tests/test_config_checker.py

```
@@ -281,6 +281,104 @@ def test_check_config_valid_ble(
+        mock_get_e2ee_status.return_value = {
+            "overall_status": "ready",
+            "enabled": True,
+            "available": True,
+            "configured": True,
+            "issues": [],
+        }
+
+        with patch("mmrelay.cli._validate_credentials_json", return_value=False):
+            result = check_config()
+
+        self.assertFalse(result)
```

Ruff (0.15.9)

[warning] Use a regular `assert` instead of unittest-style `assertFalse`

Replace `assertFalse(...)` with `assert ...`

(PT009)